app/src/features/surveys/view/summary-results/SurveySummaryResults.tsx

```diff
@@ -135,13 +135,20 @@ const SurveySummaryResults = () => {
         renderButton={(buttonProps) => {
           const { disabled, ...rest } = buttonProps;
 
-          return (
-            <ProjectRoleGuard
-              validProjectRoles={[PROJECT_ROLE.PROJECT_LEAD, PROJECT_ROLE.PROJECT_EDITOR]}
-              validSystemRoles={[SYSTEM_ROLE.SYSTEM_ADMIN, SYSTEM_ROLE.DATA_ADMINISTRATOR]}>
-              <Button {...rest} disabled={disabled || summarySubmissionStatus === PublishStatus.SUBMITTED} />
-            </ProjectRoleGuard>
-          );
+          // admins should always see this button
+          if (
+            HasProjectorSystemRole({
+              validProjectRoles: [],
+              validSystemRoles: [SYSTEM_ROLE.SYSTEM_ADMIN, SYSTEM_ROLE.DATA_ADMINISTRATOR]
+            })
+          ) {
+            return <Button {...rest} />;
+          } else {
+            // button should only be visible if the data has not been published
+            if (summarySubmissionStatus !== PublishStatus.SUBMITTED) {
+              <Button {...rest} />;
+            }
+          }
```

Comment:
Not sure if its better or worse, but would it be useful to combine the if else since they both return the same thing.

if (role OR publish) { 
  return <button/>
}

Reply:
Same in the other file